### PR TITLE
[FIX] purchase_batch_invoicing: fix visibility of invoice_status field

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa TT22179

### CODE REVIEW

**Comment by pedrobaeza** [comment]
I remember that it has also about column visibility in the context

**Comment by ernestotejeda** [comment]
Yes, but that was in v11 https://github.com/odoo/odoo/blob/11.0/addons/purchase/views/purchase_views.xml#L401
In v12 it has just invisible="1" https://github.com/odoo/odoo/blob/b405da2d3fbdf2dfc881877fd4663f0a40aa6bd6/addons/purchase/views/purchase_views.xml#L375

File: purchase_batch_invoicing/views/purchase_order_view.xml [hunk 1]
```diff
@@ -25,7 +25,7 @@
         <field name="inherit_id" ref="purchase.purchase_order_tree"/>
         <field name="arch" type="xml">
             <xpath expr="//field[@name='invoice_status']" position="attributes">
-                <attribute name="attrs"/>
+                <attribute name="invisible"/>
```
Comment (pedrobaeza) [inline purchase_batch_invoicing/views/purchase_order_view.xml]: Is this working? Shouldn't be:

```suggestion
                <attribute name="invisible">0</attribute>
```
?

Comment (ernestotejeda) [reply purchase_batch_invoicing/views/purchase_order_view.xml]: I tested it locally and it worked, but to be sure let do it as you stated, ;-)

Comment (pedrobaeza) [reply purchase_batch_invoicing/views/purchase_order_view.xml]: Well, if it was working no need of changing it. I didn't know for sure. That's why I asked. Any way, we both know for the next time.

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Fast-tracking this as obvious.

**Pull request merged**